lib/mojito/conn.ex

```diff
@@ -21,6 +21,19 @@ defmodule Mojito.Conn do
     end
   end
 
+  @doc ~S"""
+  Closes a connection
+  """
+  @spec close(t) :: :ok
+  def close(conn) do
+    # mint returns an updated conn
+    # but i don't understand why anyone
+    # would care about it. i think they just
+    # close the socket and set state: closed
```

Comment:
I think those comment are not necessary

Reply:
The docs do indeed say "you can throw away a closed connection" 

https://hexdocs.pm/mint/Mint.HTTP.html#close/1

@bmteller if you don't mind removing this comment, I'll get this merged in. Thank you for the bug fix!